apps/docs/content/docs/editor.mdx

```diff
@@ -694,6 +694,42 @@ editor.run(
 )
 ```
 
+### Hiding shapes
+
+You can pass a `isShapeHidden` predicate when configuring tldraw. This predicate will be called for each shape to determine if it should be hidden.
+
+Hidden shapes will still be present in the store, but
+
+- They will not be shown on the canvas
+- They will not be included in hit test results via `editor.getShapeAtPoint` and `editor.getShapesAtPoint`
+- They will be excluded from the arrays returned by `editor.getRenderingShapes` and `editor.getCurrentPageRenderingShapesSorted`
+- They will not be exported in image exports or rendered when printing.
+
+Otherwise, they will behave as normal.
+
+One place where this might be problematic for you, depending on what features you're using `isShapeHidden` to implement, is that hidden shapes will still be selectable, e.g. via the `select-all` action (`Cmd+A` or `Ctrl+A`).
+
+You can prevent that by cleaning up the selection in a signal reaction that is set up in the `onMount` callback:
+
+```tsx
+<Tldraw
+	isShapeHidden={(shape) => shape.meta.hidden}
+	onMount={(editor) => {
+		// We don't prevent hidden shapes from being selected out of the box, because there are some situations where it's desirable.
+		// If you want to prevent hidden shapes from being selected, you can do so like this:
+		return react('clean up selection', () => {
```

Comment:
It would be simple to add that prop, but to me it feels like a little too much to add it to the top-level tldraw props.

Plus I suspect that there will be unexpected ways in which hidden shapes conflict with other interactions, and this section of the docs pushes people towards figuring those things out on their own.